karaf/features/src/main/feature/feature.xml

```diff
@@ -55,7 +55,12 @@
         <bundle>mvn:javax.annotation/javax.annotation-api/${cxf.javax.annotation-api.version}</bundle>
     </feature>
 
+    <feature name="brooklyn-config" version="${project.version}">
```

Comment:
We could but I found it cleaner like this. Also, we may have more than one configuration features like a `brooklyn-dist-config`, `jetty-config`, etc.

With this structure, we can just add it to `brooklyn-config`